src/init/init.c

```diff
@@ -31,6 +31,28 @@ struct sof *sof_get(void)
 	return &sof;
 }
 
+#ifndef CONFIG_HAVE_SLAVE_CORE_ROM
+static inline void lp_sram_unpack(void)
```

Comment:
What is this doing. Please add descriptive comments.

Reply:
added description with AltBootManifest structure